main.py

```diff
@@ -234,36 +248,53 @@ async def execute_opportunities_webhook(webhook_url, message):
         print(f"Failed to send webhook message. Status Code: {response.status_code}")
 
 
-async def main():
-    rapid_data = opps.gpt_job_analyzer(request_rapidapi_indeed_data())
-    linkedin_data = opps.gpt_job_analyzer(request_linkedin_data())
+# async def main():
+#     # job_opps = utils.merge_all_opportunity_data(
+#     #     request_rapidapi_indeed_data(), request_linkedin_data()
+#     # )
+#     # filtered_job_opps = gpt_utils.gpt_job_analyzer(job_opps)
+#     # opps.ingest_opportunities(filtered_job_opps, "jobs_table")
```

Comment:
Additionally, leave a comment about how `filtered_job_opps` is for updating the "jobs_table" while the `filtered_internship_opps` is intended for the "internship_table" if we are to move forward with this database design.

Reply:
Ended up adding new type column and changed table name to `opportunities_table`.